modules/distr-tracing-install-otel-operator.adoc

```diff
@@ -52,3 +52,12 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
 . Navigate to *Operators* -> *Installed Operators*.
 
 . On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {OTELName} Operator shows a status of "Succeeded" before continuing.
+
+.Verification
+
+. Verify that the `status.phase` of  OpenTelemetry collector pod is `Running` and the `conditions` are `type: Ready`.
```

Comment:
@pavolloffay, verify where exactly?

Reply:
The verification should follow after collector CR installation which is missing in this document.

Reply:
What do you want me to do now? Merge this PR or wait until (I assume) you add the collector CR installation step(s)?

Reply:
It was not on my list but I can add section for it as well

Reply:
Or should I just copy the lines from https://63349--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing to this file?